packages/appstore/src/components/containers/__tests__/wallets.spec.tsx

```diff
@@ -59,21 +59,6 @@ describe('<Wallets />', () => {
         expect(container.childNodes[0]).toHaveClass('wallet__demo');
     });
 
-    it('Check content appears after arrow button click', async () => {
```

Comment:
@thisyahlen-deriv I think it's a bad practice to delete test cases if it stops to pass after refactor, you have to repair it. You can use my code:

`it('Check content appears after arrow button click', async () => {
        const Wrapper = () => {
            const [is_open, wrapperSetIsOpen] = React.useState(false);

            return (
                <StoreProvider store={mockedRootStore}>
                    <Wallet
                        wallet_account={wallet_account}
                        is_open_wallet={is_open}
                        setIsOpenWallet={wrapperSetIsOpen}
                    />
                </StoreProvider>
            );
        };

        render(<Wrapper />);

        const btn_arrow = screen.getByTestId('dt_arrow');
        expect(screen.queryByText('wallet test content')).not.toBeInTheDocument();
        expect(btn_arrow).toBeInTheDocument();

        await userEvent.click(btn_arrow);
        expect(screen.queryByText('wallet test content')).toBeInTheDocument();
    });`

Reply:
This test case doesn't make much sense 👀 We are testing the child through the parent, the child should have its own test.

Reply:
@farzin-deriv @thisyahlen-deriv ok, you can ignore my comment

Reply:
@farzin-deriv Honestly, I was thinking a little bit and I'm sure I'm right and we have to test it, because we have a CSSTransition wrapper above WalletContent and we need to test how it works. We have to test it's hidden and how it appears after arrow click. But, as you always say, you can ignore my comment

Reply:
@sergei-deriv I believe @thisyahlen-deriv already made the changes and fixed the test, Let's move this PR forward as it's not really in the scope. 

If you guys want we can have a call later and discuss this, The problem is the relation between the components we have developed, Sibling vs children, And that is leading us to issues while testing and also prop drilling. We can discuss and improve things if you guys are interested 🙇🏻